dorado/routines.py

```diff
@@ -949,3 +943,193 @@ def snake_plots(grid,
                     +os.sep+'snakefig'+str(ii)+'.png',
                     bbox_inches='tight')
         plt.close()
+
+
+def show_nourishment_area(visit_freq, grid=None, walk_data=None,
+                          cmap='Reds', sigma=0.7, min_alpha=0,
+                          show_seed=True, seed_color='dodgerblue'):
+    """Plot a smoothed, normalized heatmap of particle visit frequency
+
+    Function will plot the history of particle travel locations in walk_data
+    as a heatmap overtop the specified grid, using the output of
+    particle_track.nourishment_area(). Colors indicate number of instances
```

Comment:
Could be:
```
:obj:`dorado.particle_track.nourishment_area()`
```
if you want this text to link to the relevant function - same for below references to other functions as well as the reference to `nourishment_time()` in `show_nourishment_time()`